controllers/loadtest_controller.go

```diff
@@ -73,7 +72,29 @@ func (r *LoadTestReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	loadTestTTL := time.Duration(rawTest.Spec.TTLSeconds) * time.Second
+	loadTestTimeout := time.Duration(rawTest.Spec.TimeoutSeconds) * time.Second
```

Comment:
Great conversion.

nit: can we call this ttl and timeout or testTTL and testTimeout? We dropped the "load" from the test in a previous cleanup, because this is the LoadTestReconciler. The type of test is implied.

Reply:
I wanted to use ttl and timeout originally, but I found that in the same func we have a Timeout for reconciler (https://github.com/grpc/test-infra/blob/5c7cfabd47b59d9f7c35494be247b31414a2d431/controllers/loadtest_controller.go#L45), I will change them to testTTL and testTimeout, that is no problem :) Thank you for coming up names.